File: src/main/java/se/ifmo/pepe/dto/UserDataDTO.java
package se.ifmo.pepe.dto;

import java.util.List;

import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import se.ifmo.pepe.model.Role;
@Data
public class UserDataDTO {
  
  @ApiModelProperty(position = 0)
  private String username;
  @ApiModelProperty(position = 1)
  private String password;
  @ApiModelProperty(position = 2)
  List<Role> roles;

}

File: src/main/java/se/ifmo/pepe/dto/UserDataDTO.kt
package se.ifmo.pepe.dto

import io.swagger.annotations.ApiModelProperty
import se.ifmo.pepe.domain.Role

data class UserDataDTO(
    @ApiModelProperty(position = 0)
    val username: String,
    @ApiModelProperty(position = 1)
    val password: String,
    @ApiModelProperty(position = 2)
    val roles: List<Role>
)
